modules/installation-aws-config-yaml.adoc

```diff
@@ -73,12 +73,19 @@ platform:
       adminContact: jdoe
       costCenter: 7536
 pullSecret: '{"auths": ...}' <1>
-sshKey: ssh-ed25519 AAAA... <3>
+sshKey: ssh-ed25519 AAAA... <5>
 ----
 <1> Required. The installation program prompts you for this value.
 <2> If you do not provide these parameters and values, the installation program
 provides the default value.
-<3> You can optionally provide the `sshKey` value that you use to access the
-machines in your cluster.
+<3> The `controlplane` section is a singlet, and the `compute` section is an
+array. Because the sections have different definitions, the first line of the
+`compute` section must begin with a hyphen, `-`, and the first line of the
+`controlplane` section must not. Although both sections currently define a
+single machine pool, it is possible that future versions of {product-title}
+will support defining multiple compute pools during installation. Only one
+control plane pool is used in the current system design.
 <4> To configure faster storage for etcd, especially for larger clusters, set the
 storage type as `io1` and set `iops` to `2000`.
+<5> You can optionally provide the `sshKey` value that you use to access the
```

Comment:
Strengthen to "you need to set this if you want to debug several classes of install failures" or some such.

Reply:
I'll add it to https://github.com/openshift/openshift-docs/pull/14766/.